modules/errors.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Doesn't seem to be needed anymore? Do remove it if its not.
I don't see any use for this for now.